content/ai/concepts/neural-networks/terms/stochastic-gradient-descent/stochastic-gradient-descent.md

```diff
@@ -0,0 +1,61 @@
+---
+Title: 'Stochastic Gradient Desent'
+Description: 'Stochastic Gradient Desent is optimizer algorithm that minimizes the loss functions in machine learning and deep learning models.'
+Subjects:
+  - 'Machine Learning'
+  - 'Deep Learning'
+  - 'Computer Science'
+Tags:
+  - 'AI'
+  - 'Neural Network'
+  - 'Optimizer'
```

Comment:
```suggestion
  - 'AI'
  - 'Neural Network'
  - 'Optimizer'
```
Include only those tags that are part of [tags.md](documentation/tags.md) list. Here `optimizer` is not part of the tags list so you can include it in the tags.md list so you can use this in your PR.